examples/tf/trpo_swimmer_ray_sampler.py

```diff
@@ -6,28 +6,39 @@
 Here it runs Swimmer-v2 environment with 40 iterations.
 """
 import gym
+import ray
 
-from garage.experiment import run_experiment
+from garage import wrap_experiment
+from garage.experiment.deterministic import set_seed
 from garage.np.baselines import LinearFeatureBaseline
 from garage.sampler import RaySampler
 from garage.tf.algos import TRPO
 from garage.tf.envs import TfEnv
 from garage.tf.experiment import LocalTFRunner
 from garage.tf.policies import GaussianMLPPolicy
 
-seed = 100
 
-
-def run_task(snapshot_config, *_):
-    """Run task.
+@wrap_experiment
+def tf_trpo_swimmer(ctxt=None, seed=1):
+    """tf_trpo_swimmer.
 
     Args:
-        snapshot_config (garage.experiment.SnapshotConfig): Configuration
-            values for snapshotting.
-        *_ (object): Hyperparameters (unused).
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+
 
     """
-    with LocalTFRunner(snapshot_config=snapshot_config) as runner:
+    # Since this is an example, we are running ray in a reduced state.
```

Comment:
can you explain what is reduced about this?

Reply:
the memory and object story memory available to ray in this example is the minimum amount possible. The runs perfectly fine in this state, as it doesn't use much memory anyways.